selfdrive/ui/qt/setup/installer.cc

```diff
@@ -23,19 +23,19 @@
 #define BRANCH "master"
 #endif
 
+#ifdef MASTER
```

Comment:
let's not worry about this right now. let's just make sure it works well on release2